Server/routes/gw2.js

```diff
@@ -25,18 +25,10 @@ router.get('/members', async (req, res) => {
 
     const transformed = await Promise.all(
       data.map(async (m) => {
-        let record = await GuildMember.findOne({ memberId: m.name });
-
-        if (!record) {
-          const toSave = new GuildMember({
-            memberId: m.name,
-            eventsAttended: 0,
-          });
-          record = await toSave.save();
-        }
-
-        const eventsAttended = record ? record.eventsAttended : 0;
-        return { ...m, eventsAttended };
+        return GuildMember.findOneOrCreate(
```

Comment:
This needs to return the original "m" object plus the eventsAttended from the record:
```
const record = await GuildMember.findOneOrCreate(
  { memberId: m.name },
  { memberId: m.name, eventsAttended: 0 }
);
return { ...m, eventsAttended: record.eventsAttended };
```